include/caf/events/module_state_event.h

```diff
@@ -90,12 +113,13 @@ static inline bool module_flags_check_zero(const struct module_flags *mf)
 }
 
 /**
- * @brief Test single module bit
+ * @brief Test single module bit.
+ *
+ * @param[in] mf         Pointer to module flags variable.
```

Comment:
`module_flags`? Or "Pointer to the variable for the flags module."?

Reply:
This is actually pointer to variable that is used to store application module flags related to something (for example power management restrictions). I think that changing this to your sentence could be a bit misleading.

If you wish, I could change it to "Pointer to variable representing module flags".